ios/sandwich/QonversionSandwich.swift

```diff
@@ -13,53 +13,53 @@ public class QonversionSandwich : NSObject {
   
   // MARK: - Private variables
   
-  private var qonversionEventListener: QonversionEventListener
+  private var qonversionEventListener: QonversionEventListener? = nil
   private var isSubscribedOnAsyncEvents: Bool = false
   private var promoPurchasesExecutionBlocks = [String: Qonversion.PromoPurchaseCompletionHandler]()
   
   // MARK: - Public Functions
   
   // MARK: Initialization
   
-  public init(qonversionEventListener: QonversionEventListener) {
+  @objc public init(qonversionEventListener: QonversionEventListener) {
     self.qonversionEventListener = qonversionEventListener
   }
   
-  public func launch(
-    with projectKey: String,
+  @objc public func launch(
+    projectKey: String,
     completion: @escaping BridgeCompletion
   ) {
     Qonversion.launch(withKey: projectKey) { launchResult, error in
       if let error = error as NSError? {
         return completion(nil, error.toSandwichError())
       }
       
-      let resultDict: BridgeData = launchResult.toMap()
+      let bridgeData: [String: Any] = launchResult.toMap().clearEmptyValues()
```

Comment:
Can we use `BridgeData` here?

Reply:
Don't we need to clear empty values every time we return some values to callers?